### DISCUSSION TITLE
Louvered HVAC Vent

### DISCUSSION THREAD

**Discussion by ShehabGamrah** [firemodels/fds#13119] [Q&A]
I have a modelled jet fan and I would like the fan to blow towards this point tilted 20 degrees downards. How can I calcualte the xyz vector components.

![image](https://github.com/firemodels/fds/assets/96638521/9cb35f3f-b72f-4b6b-ab17-0caaa4a48337)
![image](https://github.com/firemodels/fds/assets/96638521/e5131c73-1f9f-485d-928f-c69b40319e29)

**Comment by drjfloyd** [comment]
![image](https://github.com/firemodels/fds/assets/12799217/63d73545-ab26-49d6-aa99-4414b24339e1)

**Comment by obscureed** [comment]
Hopefully you can work out for yourself how to calculate the x,y,x components using sines and cosines, and then work out from the FDS User Guide (as shown by @drjfloyd) how to apply it in FDS input. I would probably recommend the HVAC route, because jet fans draw in gas as well as expelling it. You should include the intake flow as well as the expelled jet (unless your fans really do inject new gas ducted in from elsewhere). With an HVAC connection between the inlet and the outlet, you can transfer the properties (such as temperature and species mass fractions) from one to the other.

I would definitely advise that you try it out on a small model with brief duration, to check that you've calculated the components correctly and that the settings have the intended effect. If your fan outlet is a single pixel, you might find that the angled UVW does not take effect at all. My experience is that it requires at least 2 × 2 pixels. (I am not sure where, if anywhere, this is stated in the documentation.)

With an outlet surface that is at least 2 × 2 cell dimensions, you will probably find that it is impossible to have the correct area of the outlet surface, compared to reality. (Even with other sizes, the area may be inaccurate.) Then you are faced with a dilemma: if you specify the velocity to achieve the correct fan flowrate, it will arrive with the wrong momentum rate (also known as thrust). If you achieve the correct thrust, the flowrate will be incorrect. This is unfortunate, but it is a consequence of realistic cell resolutions. (Typically, even 1×1 cell resolution can be too much area compared to an induction fan's outlet nozzle. So a correct flowrate will cause too little thrust; or a correct thrust will cause too much flowrate.) I believe that the better option is to achieve the correct thrust. In reality, the jet leaving the fan will quickly entrain surrounding air, becoming larger in jet width, larger in flowrate, and possibly more circular, but not too different in momentum or direction. Therefore, an oversized &VENT surface is a flawed representation of the jet emerging at the fan's nozzle, but not a bad representation of the jet slightly downstream.

**Reply by ShehabGamrah** [reply]
Thanks. I am using an HVAC inlet and outlet connected ducts to recirculate air, with a 2x2 pixel. After Specifying the vector components, the vector seems offset as shown in the picture below. How are the vector components calculated accurately.
Also should I provide vector components for the inlet vent.
![image](https://github.com/firemodels/fds/assets/96638521/d13ee428-af85-4d49-90b4-c44e56c9c273)

**Reply by obscureed** [reply]
I agree, those results do not look correct. I can only advise you to check your calculations, and run one fan at a time in an open space (just to be sure that the angled walls are not disrupting the specified jet. I would not expect the walls as shown to deflect the jet as much as your image suggests). With a small test model, you can build up complexity in stages: first achieve a slightly downward-pointing jet, in (for example) positive y direction. (This case is easier to postprocess with a slice at constant x.) Then angle it slightly in the z-axis (5 or 10 degrees, maybe). Only then angle it fully. Since you are trying to angle the jet at nearly 45 degrees (not quite, but nearly), it might be helpful to experiment with two angled vent surfaces, one x-aligned and the other y-aligned. If these both have 2×2 pixels, they represent even more area, unfortunately. Maybe it is becoming clear that an angled jet fan is not a simple thing to model in FDS -- you should expect to do some testing and development.

For the inlet side, I do not think it is required to specify vectors, and it may not be helpful or effective. The air will be pulled towards the inlet from all directions anyway.
